.gitignore

```diff
@@ -22,7 +22,8 @@ Makefile
 !conf/*
 
 CMakeCache.txt
-obj/
+obj-release/
+obj-build/
 build/
```

Comment:
I think we should also keep ignoring the old folder (obj) for those using the old handling.